UefiDbgExt/readme.md

```diff
@@ -0,0 +1,49 @@
+
+# UEFI Debugger Extension
+
+This folder contains the source for the UEFI debugger extension. This provides
+functionality within windbg for debugging the UEFI environment. Using the UEFI
+extension requires that windbgx has access to the symbol files for the target
+UEFI code.
+
+## Compiling
+
+Windbg debugger extensions need to be compiled with the Microsoft build tools.
+The easiest way to do this is to use the Developer Command Prompt that comes
+with the Visual Studio tools installation. In the command prompt, navigate to
+the folder and run "msbuild".
+
+    msbuild -property:Configuration=Release -property:Platform=x64
+
+The project can also be loaded and built in Visual Studio using the solution
+file. This project requires the Windows SDK and the Windows Driver Kit.
+
+## Installing the Extension
+
+Debugger extensions can be loaded into windbg several ways. First, by
+manually loading once already in windbg. This can be done with the .load
+command. Though this will not persist across windbg sessions.
+
+    .load <path to uefiext.dll>
```

Comment:
Do we publish this as an artifact? If so, should we add a url to point to where they are?

Reply:
Its uploaded as an artifact of the github action. which unfortunately cannot be linked in a direct way for the most recent version that I have found yes, but i'll add this information tot he readme.  e.g. https://github.com/cfernald/mu_tiano_platforms/actions/runs/3371109346

Reply:
after thinking a bit more, we will probably just publish it along with the firmware (probably to nuget) when we setup the release pipeline.